src/marshmallow_annotations/converter.py

```diff
@@ -100,3 +109,10 @@ def _get_type_hints(self, item):
         for parent in item.__mro__[::-1]:
             hints.update(get_type_hints(parent))
         return hints
+
+    def _get_field_defaults(self, item):
+        """Retrieve default values if item is a namedtuple."""
+        if hasattr(item, '_field_defaults'):
```

Comment:
double quotes.

Reply:
Additionally, I don't think this belongs in the base:

```python
class SomeThing:
    a: int = 1

hasattr(SomeThing, '_field_defaults') # False
```

If we do put it here, the default implementation should be:

```python
def _get_field_defaults(self, target):
    return {}
```

And then each extension can provide their own implementation if needed. We'd have to document it as a non-public override point but that's not uncommon even in stricter OO languages.